lib/optimizely/helpers/constants.rb

```diff
@@ -306,6 +306,12 @@ module Constants
         'INTEGER' => 'integer',
         'STRING' => 'string'
       }.freeze
+
+      INPUT_VARIABLES = {
+        'user_id' => 'User ID',
```

Comment:
If you look at `VARIABLE_TYPES` the keys are all caps. Let's keep it consistent here.